### PR TITLE
Create card short names for looking up e.g. 'bob' 'steve'

### PR DESCRIPTION
**Pull request by Villawhatever**
#120

### CODE REVIEW

File: short_names.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "shortname": "bob",
+    "fullname": "Dark Confidant"
+  }, {
+    "shortname": "baby jace",
+    "fullname": "Jace, Vryn's Prodigy"
+  },
+  {
+    "shortname": "jtms",
+    "fullname": "Jace, the Mind Sculptor"
+  },
```
Comment (Fryyyyy) [inline short_names.json]: Inconsistent formatting in the file here "}, {" vs "}, \n {"

File: card.go [hunk 2]
```diff
@@ -476,6 +476,13 @@ func getCachedOrStoreCard(card *Card, ncn string) (Card, error) {
 func getScryfallCard(input string) (Card, error) {
 	cardRequests.Add(1)
 	var card Card
+	
+	// See if it's a known short name of a card.
+	if qualifiedName, ok := shortCardNames[input]; ok {
```
Comment (Fryyyyy) [inline card.go]: Since we're not normalising here does that mean "tnn" won't match "TNN" etc?

File: short_names.json (hunk 1 above)
Comment (Villawhatever) [reply short_names.json]: Fixed. Former over latter, removed errant newlines.

File: card.go (hunk 2 above)
Comment (Villawhatever) [reply card.go]: Fixed. Moved normalisation above, verified all names in short_names.json are lowercase.

**Pull request merged**